src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -157,10 +157,37 @@ export default {
   },
 
   async mounted() {
-    await this.openStream();
-    await this.initRecorder();
-    if (this.recorderState === RECORDER_STATE.INITIALIZING) {
-      await this.startRecording();
+    try {
+      await this.openStream();
+      await this.initRecorder();
+
+      if (this.recorderState === RECORDER_STATE.INITIALIZING) {
+        await this.startRecording();
+      }
+    } catch (err) {
+      let errorMessage;
+
+      switch (err.name) {
+        case 'NotAllowedError':
+          errorMessage = this.$t(
+            'components.uploadVideo.recorder.errors.cameraNotAllowed',
+          );
+          break;
+
+        case 'NotFoundError':
+          errorMessage = this.$t(
+            'components.uploadVideo.recorder.errors.cameraNotFound',
+          );
+          break;
+
+        default:
+          errorMessage = this.$t(
+            'components.uploadVideo.recorder.errors.default',
+          );
+          break;
+      }
+
+      this.$emit('error', errorMessage);
```

Comment:
and if user try to recall init video recording, error will stay, but it should leave, if all is ok

Reply:
done as well

please check following scenario
- allow camera
- wait when recorder will be initialized, then revoke permissions without page reload (ff allows it)
- you should be able to allow camera again (earlier you just see continue button)